Caltech-IPAC/firefly
# Firefly-75: Implement new WCS matching scheme

Firefly-75: Implement new WCS matching scheme

  - WCS, Target, unlock match working from dropdown
  - refactored locking to break up overlay/color and position
  - HiPS working
  - pixel matching working
  - pixel center matching
  - multiple icons
  - fixed no WCS bug
  - cleaned out old UI
  - WCS match type is a startup option

_To test:_
https://irsawebdev9.ipac.caltech.edu/firefly-75-wcsmatch/firefly/

- Load several types of images
- Choose a match style from the dropdown: ![28x28_Match_Unlocked](https://user-images.githubusercontent.com/6730519/59311026-5b213800-8c65-11e9-9a63-eb683e63e137.png)
- Notice:
   - 4 matches that dow't lock
   - 4 locking matches
   - unlocking
- Test separate color/overlay lock: ![OverlayUnlocked](https://user-images.githubusercontent.com/6730519/59311159-bd7a3880-8c65-11e9-9736-8c66564e42cc.png)
- Notice:
    - when locked color changes together
    - when locked overlay change together (try distance tool)

You are welcome to give feedback on the icons but remember we only have limited skill to create new icons the match the current style.
